### DISCUSSION TITLE
Bug: "All Activity" is displayed on the user page

### DISCUSSION THREAD

**Discussion by Hiratake** [community/community#12544] [Other Feature Feedback, Questions, & Ideas]
Please see this GIF.
Click the "organization" that made the contribution, then click the same location again.  
Then, "All Activity" that should not be displayed is displayed.

![github](https://user-images.githubusercontent.com/23224932/157253366-00502d85-5619-48c1-a9bd-8b4f74b25ee4.gif)

This phenomenon does not seem to occur when you are not logged in. 

The executed environment is as follows. 
- Windows 10
- Firefox 98.0b10 (64bit)

The same thing happens with Chrome. 

I'm not good at English. I'm sorry if it doesn't make sense.

**Comment by book000** [comment]
Hello. I also found this bug.

The network communication log shows that three request are occurring when the selection is deselected.

- `https://github.com/?from=NaN-aN-aN&to=NaN-aN-aN#%3CPromise:0x00007f7a934b87e0%3E`
- `https://github.com/book000?from=2022-03-01&to=2022-03-08`
- `https://github.com/book000?from=2022-03-01&to=2022-03-08&year_list=1`

It appears that the first of these communication responses has been displayed as is.  
This request appears to have been issued by `https://github.githubassets.com/assets/github_behaviors_pjax_ts-cb5bd859c0a81724eff3.js`.

I hope this helps fix the bug!
